packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -103,6 +105,15 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           '&:active, &:hover, &:active:hover': {
             color: linkInteractedColor,
             textDecoration: 'underline',
+
+            selectors: {
+              [HighContrastSelectorBlack]: {
+                color: '#FFFF00',
+              },
+              [HighContrastSelectorWhite]: {
+                color: '#00009F',
+              },
+            },
```

Comment:
hm, are these needed?

Reply:
If we don't do this the styling when we hover over them looks really wrong.

Reply:
my bad, i thought these are added for `!isButton &&` case :)